### PR TITLE
SLSA Provenance generation blog post

### PR DESCRIPTION
**Pull request by developer-guy**
kindly ping @Dentrax

### CODE REVIEW

File: www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+---
+date: 2023-10-08
+slug: slsa-generation-for-your-artifacts
+categories:
+  - announcements
```
Comment (caarlos0) [inline www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md:5]: maybe add a new category: `tutorials`? (or `guides` or something else)

File: www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
...has never been greater. Organizations must take steps to protect their software development and distribution processes from potential vulnerabilities and attacks. SLSA provenance, short for Supply Chain Levels for Software Artifacts, is an emerging concept that revolutionizes the way we think about software supply chain security. It offers a comprehensive framework to track the lineage and trustworthiness of software components, thereby enhancing overall security. The core idea behind SLSA provenance is to create a transparent and auditable trail of every software component's journey, from its creation to deployment. This ensures that any tampering or unauthorized changes can be quickly identified and mitigated. Software supply chain security and SLSA provenance are intrinsically linked, as the latter serves as a critical tool to bolster the former. Together, they provide a robust defense against the growing threats posed by malicious actors in the digital realm. In a world where software vulnerabilities can have far-reaching consequences, the adoption of SLSA provenance is a proactive step toward fortifying our software supply chains and making them more resilient to cyberattacks.
```
Comment (caarlos0) [inline www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md:13]: this paragraph felt too big to me, can we shorten it somehow? or maybe split into multiple paragraphs?

File: www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+In this blog post, we will explore how GoReleaser can help you generate SLSA provenance for your artifacts and how you can leverage the slsa-github-generator to automate the process.
+
+## slsa-github-generator
+
+I would like to start with my favorite quote:
+
+> _Each of these attacks could have been prevented if there were a way to detect that the delivered artifacts diverged from the expected origin of the software. But until now, generating verifiable information that described where, when, and how software artifacts were produced (information known as provenance) was difficult. This information allows users to trace artifacts verifiably back to the source and develop risk-based policies around what they consume._ - [Improving software supply chain security with tamper-proof builds](https://security.googleblog.com/2022/04/improving-software-supply-chain.html)
+
+Currently (_unfortunately_), provenance generation is not widely supported yet but hopefully will be in the future. And this is where the slsa-github-generator comes into play.
```
Comment (caarlos0) [inline www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md:27]: I would phrase it as `Unfortunately, provenance generation is not widely supported yet, but hopefully will be in the future` or something like that

File: www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+Currently (_unfortunately_), provenance generation is not widely supported yet but hopefully will be in the future. And this is where the slsa-github-generator comes into play. 
+
+Thankfully to the SLSA community, they developed a collection of GitHub reusable workflows called [slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator) that can help us with the solving of the problem we mentioned in the quote. It is a powerful tool designed to simplify the process of generating SLSA provenances for your GitHub-hosted projects. It seamlessly integrates with your GitHub repositories, providing an efficient way to enhance the security and trustworthiness of your software supply chain by automatically creating and managing SLSA provenance records.
```
Comment (caarlos0) [inline www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md:29]: "Thanks to the SLSA community" maybe?

**Review by caarlos0 — COMMENTED** [review]
awesome post, made a few comments, most of them cosmetic, feel free to dismiss them if you disagree.

thanks for the submission btw 🙏

File: www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md (hunk 2 above)
Comment (developer-guy) [reply www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md:13]: okay definitely makes sense

**Review by caarlos0 — APPROVED** [review]

File: www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
+---
+date: 2023-10-08
```
Comment (caarlos0) [inline www/docs/blog/posts/2023-10-08-slsa-generation-for-your-artifacts.md:2]: just need to update the date here and in the filename, and IMHO we can merge this :D

**Comment by developer-guy** [comment]
there are some issues in the docs/htmltest job but don't know how to fix them PTAL @caarlos0

**Comment by caarlos0** [comment]
> there are some issues in the docs/htmltest job but don't know how to fix them PTAL @caarlos0

those are very brittle, I should fix them at some point

**Pull request merged**